### DISCUSSION TITLE
Add Alphas to GPlay Beta and/or F-Droid Repos

### DISCUSSION THREAD

**Discussion by WingofaGriffin** [LawnchairLauncher/lawnchair#3180] [General]
It feels silly that there are live entries for Lawnchair on these marketplaces, yet the version of Lawnchair hosted is so out of date. I think taking advantage of the Play Store Beta channel and the F-Droid tools to release the Alphas in a more streamlined manner than Discord, Telegraph, and GitHub is a smart idea for accessibility and keeping versions updated in a more simple manner than manual APK installation. This way the "stable" version still exists, but the Alpha can still be used in more intuitive places.

**Comment by jeffdumcum** [comment]
I completely agree

**Comment by tungstengmd** [comment]
Perfect idea.

**Comment by SuperDragonXD** [comment]
Hello! I know that this reply is a bit late, but we now have a Play Store entry. F-Droid is unfortunately blocked due to #1473.
